contracts/test/integration/utils/RoleTesting.sol

```diff
@@ -31,6 +31,9 @@ contract RoleTesting is DSTest {
         uint256[] memory numEachRole
     ) internal view {
         for (uint256 i = 0; i < allRoles.length; i++) {
+            if (i == 2) {
```

Comment:
We should say if i == 2 and we're on the arbitrum network, then we just continue and don't verify this role.